downloads/index.md

```diff
@@ -263,27 +263,24 @@ Checksums for this release are available in both, [MD5](https://julialang-s3.jul
 ~~~
 Most Linux users should use the glibc binaries unless you know that your system uses musl as its libc.
 
-## Older Releases
+## Older releases
```

Comment:
Is there a specific reason to change these? Won't this break direct links?

Reply:
I've also deleted some unnecessary section titles. So links are going to break. But this one has been reverted.